source/future.tex

```diff
@@ -1468,8 +1468,7 @@
 only if the type \tcode{T} is any of the following:
 \begin{itemize}
 \item a function type or a pointer to function type taking one argument of type \tcode{T1}
-% FIXME: Should this be R T0::f() ?
-\item a pointer to member function \tcode{R T0::f} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
+\item a pointer to member function \tcode{R T0::f()} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
```

Comment:
I think it is supposed to be `R T0::f()` _`cv`_, and should be read as:

> a pointer, to a member function of the form `R T0::f()` _`cv`_

i.e.

> a pointer to member function that points to a member function `R T0::f()` _`cv`_


It certainly would be clearer to talk about the type of the pointer to member function, not show a declaration of a member function.